src/contracts/BToken.sol

```diff
@@ -12,46 +12,46 @@ contract BToken is ERC20 {
 
   /**
    * @notice Increase the allowance of the spender.
-   * @param dst The address which will spend the funds.
-   * @param amt The amount of tokens to increase the allowance by.
-   * @return True if the operation is successful.
+   * @param spender The address which will spend the funds.
+   * @param amount The amount of tokens to increase the allowance by.
+   * @return success True if the operation is successful.
    */
-  function increaseApproval(address dst, uint256 amt) external returns (bool) {
-    _approve(msg.sender, dst, allowance(msg.sender, dst) + amt);
+  function increaseApproval(address spender, uint256 amount) external returns (bool success) {
+    _approve(msg.sender, spender, allowance(msg.sender, spender) + amount);
     return true;
```

Comment:
We are doing different types of returns (sometimes by value, others no return, etc)